<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>798</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>股票代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>740</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_num">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>690</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>740</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_history">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>当前页数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_page">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>到尾页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>到首页</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>股票名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>640</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
